doc/user_guide/storage_plugins.md

```diff
@@ -28,6 +28,37 @@ It also comes with a specific `Blob` and `Image` storage plugin, that can load a
 Storage plugins can be written in either C or Python.
 
 
+How to make storage plugins available
+-------------------------------------
+
+As described below it is possible (and most often advisable) to create specific drivers (storage plugins) for your data.
+Additional storage plugins drivers can be made available by setting the environment variables
+`DLITE_STORAGE_PLUGIN_DIRS` or `DLITE_PYTHON_STORAGE_PLUGIN_DIRS` e.g.:
+```bash
+export DLITE_STORAGE_PLUGIN_DIRS=/path/to/new/folder:$DLITE_STORAGE_PLUGIN_DIRS
+```
+
+Within python, the path to the directory containing plugins can be added as follows:
+
+```python
+import dlite
+dlite.python_storage_plugin_path.append("/path/to/plugins/dir")
+```
+
+Often drivers are connected to very specific datamodel (entities). 
+DLite will find these datamodels if the path to their directory is set with the
+environment variable `DLITE_STORAGES` or added within python with `dlite.storage_path.append` similarly to described above for drivers.
+
+
+IMPORTANT:
+Often, during development dlite will fail unexpectedly. This is typically either because of an error in the 
+datamodel or the driver. 
+The variable DLITE_PYDEBUG can be set as `export DLITE_PYDEBUG=` to get python debugging information. 
+This will give information about the driver.
+But is is advisable to first check that the datamodel is valid with the command `dlite-validate datamodelfilename`.
```

Comment:
This will be put in one line when rendered. I will suggest to use an admonition (https://sphinx-book-theme.readthedocs.io/en/stable/reference/kitchen-sink/admonitions.html).

```suggestion
```{attention}
Often, during development dlite will fail unexpectedly. This is typically either because of an error in the 
datamodel or the driver. 
The variable DLITE_PYDEBUG can be set as `export DLITE_PYDEBUG=` to get python debugging information. 
This will give information about the driver.
But is is advisable to first check that the datamodel is valid with the command `dlite-validate datamodelfilename`.
```
```